### PR TITLE
Updated configuration

### PR DESCRIPTION
**Pull request by petrdvorak**
We are releasing a new version of the dev portal and we need to adjust the search layout accordingly

### CODE REVIEW

File: configs/wultra_developers.json [hunk 1]
```diff
@@ -8,26 +8,43 @@
...
+    "lvl5": "section h4",
     "text": "section p, section li"
   },
   "selectors_exclude": [
     ".sidebar-toc",
     ".sidebar"
   ],
-  "conversation_id": [
-    "1176317311"
-  ],
```
Comment (shortcuts) [inline configs/wultra_developers.json:25]: This is a mandatory field, could you please revert it?

File: configs/wultra_developers.json [hunk 2]
```diff
@@ -8,26 +8,43 @@
...
+      "hierarchy",
+      "content",
+      "anchor",
+      "url",
+      "url_without_anchor",
+      "type"
     ]
-  },
-  "nb_hits": 26160
-}
```
Comment (shortcuts) [inline configs/wultra_developers.json:33]: This one is also mandatory

File: configs/wultra_developers.json (hunk 1 above)
Comment (petrdvorak) [reply configs/wultra_developers.json:25]: Ah... I am sorry - we disabled this for the local debugging...

Comment (shortcuts) [reply configs/wultra_developers.json:25]: No worries :D

**Pull request merged**